src/containers/views/DataFileEdit.js

```diff
@@ -120,7 +124,7 @@ export class DataFileEdit extends Component {
       return <h1>{getNotFoundMessage("data file")}</h1>;
     }
 
-    const { path, raw_content, content } = datafile;
+    const { path, raw_content, content, ext } = datafile;
```

Comment:
Already have filename below this line. For this and above comments, I think it would be better to have a `getExtension` helper.